app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "Visitor is not vaccinated."
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "Vaccine is outdated."
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return "Visitor is not wearing a mask."
```

Comment:
The problem with this approach is -> if we want to make customizable messages, like normally inherited from an exception, we won't create them, since str will always return only this defined message. Better way will be something like:
```suggestion
class NotVaccinatedError(VaccineError):
    def __init__(self, message="Visitor is not vaccinated."):
        super().__init__(message)
```